frontend/src/components/common/Input/styles.ts

```diff
@@ -0,0 +1,22 @@
+import styled from '@emotion/styled';
+
+export const Input = styled.input<{ $width?: string; $height?: string }>`
```

Comment:
Input 스타일의 props를 밖으로 쓰는게 아닌 안에 넣어준 이유가 있을까요? 
```suggestion
interface InputProps { $width?: string; $height?: string }
export const Input = styled.input<InputProps>`
```

Reply:
아직 논의된 사항은 아니긴 하지만 통일성을 위해 리터럴 대신 인터페이스를 가져와서 쓰는 걸로 수정했습니다~!
나중에 얘기해봐요!